bower.json

```diff
@@ -26,7 +26,7 @@
     "n-sliding-popup": "^1.0.0",
     "next-myft-client": "^6.4.2",
     "o-ads": "^6.0.0",
-    "o-buttons": "4.4.4",
+    "o-buttons": "^4.4.4",
```

Comment:
Wouldn't we need to force the issue to make next-article work? i.e. `o-buttons": "^4.5.0`